modules/ensemble/lib/framework/notification_manager.dart

```diff
@@ -138,12 +154,15 @@ class NotificationManager {
     } on Exception catch (e) {
       print("NotificationManager: Error receiving notification: $e");
     }
+    print("----------");
```

Comment:
either convert it to debugPrint or remove it

Reply:
Updated